Tests/ApolloCodegenTests/CodeGeneration/Templates/PersistedQueriesOperationManifestTemplateTests.swift

```diff
@@ -0,0 +1,226 @@
+import XCTest
+import Nimble
+@testable import ApolloCodegenLib
+import ApolloCodegenInternalTestHelpers
+
+class PersistedQueriesOperationManifestTemplateTests: XCTestCase {
+  var subject: PersistedQueriesOperationManifestTemplate!
+
+  override func setUp() {
+    super.setUp()
+
+    subject = PersistedQueriesOperationManifestTemplate(config: .init(config: .mock()))
+  }
+
+  override func tearDown() {
+    subject = nil
+
+    super.tearDown()
+  }
+
+  // MARK: Render tests
+
+  func test__render__givenSingleOperation_shouldOutputJSONFormat() throws {
+    // given
+    let operation = IR.Operation.mock(
+      name: "TestQuery",
+      type: .query,
+      source: """
+        query TestQuery {
+          test
+        }
+        """
+    )
+
+    let expected = """
+      {
+        "format": "apollo-persisted-queries",
```

Comment:
Should be `apollo-persisted-query-manifest`. Did you get this from some docs or other request that might need to be fixed?

Reply:
Thanks for the heads up. Yeah, I got this from the Project: Persisted Queries Quip doc, so it needs to be updated there!

Reply:
Ah, the one that says "The format will be something like the following" :)